packages/expo-secure-store/src/SecureStore.ts

```diff
@@ -131,6 +131,8 @@ export async function deleteItemAsync(
  *
  * @return A promise that resolves to the previously stored value, or `null` if there is no entry
  * for the given key. The promise will reject if an error occurred while retrieving the value.
+ *
+ * > When reading from an invalidated key the promise will return `null` on iOS, on Android it will be rejected.
```

Comment:
Let's unify the behavior here, I think returning `null` on both platforms makes more sense than throwing exceptions.